src/test/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGeneratorTest.java

```diff
@@ -240,5 +276,7 @@ private void verifyEiffelSourceChangeCreatedEvent(
         assertEquals(errorMessage, CHANGE_ID, change.get("id").getAsString());
         assertEquals(errorMessage, SIZE_INSERTIONS, change.get("insertions").getAsInt());
         assertEquals(errorMessage, SIZE_DELETIONS, change.get("deletions").getAsInt());
+        assertEquals(errorMessage, "PREVIOUS_VERSION", link1.get("type").getAsString());
+        assertEquals(errorMessage, "BASE", link2.get("type").getAsString());
```

Comment:
There are 4 cases but you are only testing 1 case

Reply:
We are now covering: no links and link population in both events in the tests.